File: ArkhamAsylum.Lib/Models/Record.cs
using Newtonsoft.Json;
using System;

namespace ArkhamAsylum.Lib.Models
{
    public class Record : Entity
    {
        public string Code { get; set; }

        public Guid PatientId { get; set; }

        [JsonIgnore]
        public Patient Patient { get; set; }

        public Guid BedId { get; set; }

        [JsonIgnore]
        public Bed Bed { get; set; }

        public Guid DiagnosisId { get; set; }

        [JsonIgnore]
        public Diagnosis Diagnosis { get; set; }

    }
}

File: ArkhamAsylum.Web/wwwroot/Scripts/Models/Record.js
class Record extends Entity
{
    constructor(json)
    {
        super(json);
        if (json)
        {
            this.Code = json.code;
            this.PatiendId = json.patientId;
            this.BedId = json.bedId;
            this.DiagnosisId = diagnosisId;
        }
        else
        {
            this.Code = "";
            this.PatiendId = "00000000-0000-0000-0000-000000000000";
            this.BedId = "00000000-0000-0000-0000-000000000000";
            this.DiagnosisId = "00000000-0000-0000-0000-000000000000";
        }
    }
}